core/src/test/scala/org/apache/spark/TempLocalSparkContext.scala

```diff
@@ -51,7 +51,7 @@ trait TempLocalSparkContext extends BeforeAndAfterEach
    */
   def sc: SparkContext = {
     if (_sc == null) {
-      _sc = new SparkContext(_conf)
+      _sc = new SparkContext(conf)
```

Comment:
isn't it the same?

Reply:
No. `conf` allow us to use the custom conf from inherit classes.  But `_conf`  are hard-coded to `defaultSparkConf`.